sonic-net/sonic-buildimage
# [ci] Disable DOCKER_BUILDKIT in pipeline build for OOM issue.

#### Why I did it
New docker release v23.0 uses BUILDKIT by default.
It leads to OOM issue in pipeline build. 
```
##[error]Exit code 137 returned from process: file name '/agent/externals/node16/bin/node',
```
#### How I did it
Disable BUILDKIT when building sonic-slave-* image.
Keep checking if there are issues when building docker image inside sonic-slave-*.
#### How to verify it
Check docker build logs.
Disable BUILDKIT log:
```
Step 1/80 : FROM publicmirror.azurecr.io/debian:buster
 ---> ff5db168d4c5
```
#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog


#### Ensure to add label/tag for the feature raised. example - PR#2174 under sonic-utilities repo. where, Generic Config and Update feature has been labelled as GCU.

#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
